### PR TITLE
More small Registration V2 Fixes

### PR DESCRIPTION
**Pull request by FinnIckler**
- Fixes Guests being errored incorrectly
- Fixed Event Selector showing error before being interacted with if no preferred event is set or a favourite competition is used
- Fixed a bunch of strings
- deleted some unused imports
- flipped the columns of the waiting list table

Needs a small PR on wca-registration to add `registration?.payment.payment_amount_human_readable`([already done, just need to open it](https://github.com/thewca/wca-registration/pull/605))

### CODE REVIEW

**Comment by dunkOnIT** [comment]
@thewca-bot deploy staging

File: config/locales/en.yml [hunk 1]
```diff
@@ -1772,8 +1772,8 @@ en:
         acknowledgement: "I have read and acknowledged all information under the 'Register' tab, including entry fees, refund policies and any extra requirements listed above."
         next_step: "Continue to next Step"
       update:
-        being_updated: "Registration is being updated"
-        being_deleted: "Registration is being deleted"
+        being_updated: "Registration is being updated..."
+        being_deleted: "Registration is being deleted..."
```
Comment (gregorbg) [inline config/locales/en.yml:1776]: How are we handling `...` ellipsis in other locale string?
I know there is a single Unicode `…` character as opposed to three `.` characters

Comment (FinnIckler) [reply config/locales/en.yml:1776]: seems to be all `...` and no `…`

File: app/webpacker/components/RegistrationsV2/Register/CompetingStep.jsx [hunk 2]
```diff
@@ -83,15 +86,27 @@ export default function CompetingStep({
...
         {
           ...data.registration,
           payment: registration.payment,
         },
       );
-      nextStep();
+      // Going from pending -> Cancelled
+      if (data.registration.competing.registration_status === 'cancelled') {
+        dispatch(setMessage('competitions.registration_v2.register.registration_status.cancelled', 'positive'));
+        nextStep({ toStart: true });
```
Comment (FinnIckler) [inline app/webpacker/components/RegistrationsV2/Register/CompetingStep.jsx:99]: I could return here to next one level deeper?

**Review by gregorbg — APPROVED** [review]

File: app/webpacker/components/RegistrationsV2/Register/CompetingStep.jsx (hunk 2 above)
Comment (gregorbg) [reply app/webpacker/components/RegistrationsV2/Register/CompetingStep.jsx]: Seems you already made up your mind :smile:

**Pull request merged**